jenkins/control-slave.sh

```diff
@@ -0,0 +1,146 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+
+DOCKER_VERSION='1.9.1-0~wheezy'
+GO_VERSION='go1.6.linux-amd64.tar.gz'
```

Comment:
This doesn't match our internal docs, and doesn't match what we are running right now.